lib/tasks/students.rake

```diff
@@ -30,4 +30,17 @@ namespace :students do
       end
     end
   end
+
```

Comment:
Migrations should not be added as rake tasks. They should be added in a migration folder. Please have a look at the new migration architecture in bibliotheca.